_includes/head.html

```diff
@@ -1,7 +1,11 @@
 <head>
   <meta charset='utf-8' />
   <meta http-equiv="X-UA-Compatible" content="chrome=1" />
-  <meta name="description" content="Up For Grabs : Jump in!" />
+  <meta property="og:url"         content="http://up-for-grabs.net" />
+  <meta property="og:type"        content="product" />
+  <meta property="og:title"       content="Up For Grabs" />
+  <meta property="og:description" content="Want to contribute to open source, but GitHub feels like an ocean? Grab these tiny issues before they dissolve themselves!" />
```

Comment:
We do have a few non-GitHub projects... "Want to contribute to open source, but not sure where to start?"

I'm also not sure exactly what "before they dissolve themselves" means?